city.lua

```diff
@@ -1524,7 +1520,8 @@ local function start_house_construction()
 
             local buildables = getBuildables(hardware_stores)
             -- If there are no hardware stores, then no construction resources are available.
-            for _, tier in ipairs(housing_tiers) do
+            -- use CITIZEN_COUNTS as housing tiers list
+            for tier, _ in pairs(Constants.CITIZEN_COUNTS) do
```

Comment:
i hope this code doesn't depend on order, because FactorioLua's order differs from Lua-5.3 (standalone)
looks like `serpent` has alphabetical sort when called from console:
```
73034.841 Script log(serpent.block({ simple=1, residential=2, highrise=3, })):1: {
  highrise = 3,
  residential = 2,
  simple = 1
}
```
 but i just checked this from migration script - order matches just as declared in `Constants`

Reply:
This code doesn't depend on order anymore. 👍